app.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+var hours = ['6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm'];
+
+var pike = {
+  name:'1st & Pike',
```

Comment:
we usually try to put a space between the "colon" and the value associated with the key.